packages/types/src/base.ts

```diff
@@ -51,6 +51,7 @@ type AbstractAsset = {
   caip2: string
   chain: ChainTypes
   description?: string
+  isTrustedDescription?: boolean | undefined
```

Comment:
```suggestion
  isTrustedDescription?: boolean
```

`boolean | undefined` means if it is defined, it can also be undefined